### DISCUSSION TITLE
Get Rank from Vault itself instead of storing separately

### DISCUSSION THREAD

**Discussion by YTRemRem** [Zrips/CMI#2002] [Feature Requests]
The rankup feature should get rank from the Vault API
---
### Detailed explanation what it should do and when and in what situations ###
Rank should be check with the vault plugin
### Basic explanation why this feature is needed in CMI and how useful it would be to have it ###
It will save storage space if your server has a lot of player, it doesn't need to save the rank data separately, instead of saving it separately, why not just pull it from the Vault plugin API? It will get the correct rank even when you set the rank without using the rankset command.

**Comment by Zrips** [comment]
Ranks are separate from players group, i'm not aware of any other record about players group/rank then list of all permissions groups he has access too and main group. Unless i'm missing something obvious here.
